docs/user/cli-troubleshooting.md

```
@@ -0,0 +1,347 @@
+If you continue to experience issues:
+
+1. **Collect diagnostic information**:
+   ```bash
+   # Get system information
+   flightctl version
+   uname -a
+   cat /etc/os-release
+   ```
+
+2. **Generate SOS report** (if applicable):
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)